distribution/openrct2.d.ts

```diff
@@ -2322,19 +2322,88 @@ declare global {
 
     enum TrackSlope {
         None = 0,
+        Up12 = 1,
```

Comment:
The additional values need to be implemented in ScriptEngine.cpp.

Reply:
Thank you for pointing this out.